tests/Resources/ti.geolocation.test.js

```diff
@@ -244,6 +246,30 @@ describe.windowsBroken('Titanium.Geolocation', function () {
 		should(Ti.Geolocation.trackSignificantLocationChange).eql(true);
 	});
 
+	it.ios('.ACCURACY_REDUCED', function () {
+		if (isiOS14) {
+			should(Ti.Geolocation).have.constant('ACCURACY_REDUCED').which.is.a.Number();
+		}
+	});
+
+	it.ios('.ACCURACY_AUTHORIZATION_FULL', function () {
+		if (isiOS14) {
+			should(Ti.Geolocation).have.constant('ACCURACY_AUTHORIZATION_FULL').which.is.a.Number();
+		}
+	});
+
+	it.ios('.ACCURACY_AUTHORIZATION_REDUCED', function () {
+		if (isiOS14) {
+			should(Ti.Geolocation).have.constant('ACCURACY_AUTHORIZATION_REDUCED').which.is.a.Number();
+		}
+	});
+
+	it.ios('.locationAccuracyAuthorization', function () {
```

Comment:
Add test that verifies `requestTemporaryFullAccuracyAuthorization` method exists (and maybe additionally that on iOS < 14 it immediately calls the callback with an error)

Reply:
Added unit test for `requestTemporaryFullAccuracyAuthorization `.